java.lang.NoSuchMethodError with 3.0.1 and Vaadin 8.2.0.beta1

Hi.

I tried to upgrade from Vaadin 8.1.6 to 8.2.0.beta1 and got the following exception when navigating to a view which contains multi file upload addon:

```2017-11-24 12:12:02,525 ERROR AbstractApplicationUI                   :    ? - A fatal error occurred caused by an uncaught exception.
com.vaadin.server.ServerRpcManager$RpcInvocationException: Unable to invoke method click in com.vaadin.shared.ui.button.ButtonServerRpc
	at com.vaadin.server.ServerRpcManager.applyInvocation(ServerRpcManager.java:159)
	at com.vaadin.server.ServerRpcManager.applyInvocation(ServerRpcManager.java:116)
	at com.vaadin.server.communication.ServerRpcHandler.handleInvocation(ServerRpcHandler.java:445)
	at com.vaadin.server.communication.ServerRpcHandler.handleInvocations(ServerRpcHandler.java:410)
	at com.vaadin.server.communication.ServerRpcHandler.handleRpc(ServerRpcHandler.java:274)
	at com.vaadin.server.communication.UidlRequestHandler.synchronizedHandleRequest(UidlRequestHandler.java:90)
	at com.vaadin.server.SynchronizedRequestHandler.handleRequest(SynchronizedRequestHandler.java:40)
	at com.vaadin.server.VaadinService.handleRequest(VaadinService.java:1588)
	at com.vaadin.server.VaadinServlet.service(VaadinServlet.java:445)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:731)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:303)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
	at com.myapp.app_base.AccessLogFilter.doFilter(AccessLogFilter.java:49)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:219)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:110)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:506)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:169)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)
	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:962)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:116)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:445)
	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1115)
	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:637)
	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:318)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.vaadin.server.ServerRpcManager.applyInvocation(ServerRpcManager.java:155)
	... 29 more
Caused by: com.vaadin.event.ListenerMethod$MethodException: Invocation of method buttonClick in com.github.appreciated.app.layout.builder.providers.AbstractNavigationElementComponentProvider$$Lambda$120/1850797107 failed.
	at com.vaadin.event.ListenerMethod.receiveEvent(ListenerMethod.java:519)
	at com.vaadin.event.EventRouter.fireEvent(EventRouter.java:273)
	at com.vaadin.event.EventRouter.fireEvent(EventRouter.java:237)
	at com.vaadin.server.AbstractClientConnector.fireEvent(AbstractClientConnector.java:1014)
	at com.vaadin.ui.Button.fireClick(Button.java:384)
	at com.vaadin.ui.Button$1.click(Button.java:57)
	... 34 more
Caused by: java.lang.NoSuchMethodError: com.vaadin.ui.Grid$Column.setDescriptionGenerator(Lcom/vaadin/ui/components/grid/DescriptionGenerator;)Lcom/vaadin/ui/Grid$Column;
	at com.wcs.wcslib.vaadin.widget.multifileupload.ui.UploadQueueGrid.<init>(UploadQueueGrid.java:39)
	at com.wcs.wcslib.vaadin.widget.multifileupload.ui.UploadStatePanel.<init>(UploadStatePanel.java:53)
	at com.wcs.wcslib.vaadin.widget.multifileupload.ui.UploadStatePanel.<init>(UploadStatePanel.java:56)
	at com.wcs.wcslib.vaadin.widget.multifileupload.ui.MultiFileUpload.createStatePanel(MultiFileUpload.java:71)
	at com.wcs.wcslib.vaadin.widget.multifileupload.ui.MultiFileUpload.<init>(MultiFileUpload.java:36)
	at com.wcs.wcslib.vaadin.widget.multifileupload.ui.MultiFileUpload.<init>(MultiFileUpload.java:41)
	at com.wcs.wcslib.vaadin.widget.multifileupload.ui.MultiFileUpload.<init>(MultiFileUpload.java:45)
	at com.myapp.frontend.ui.component.UserFileEditorTab.<init>(UserFileEditorTab.java:84)
	at com.myapp.frontend.ui.component.PersonEditorWrapper.<init>(PersonEditorWrapper.java:79)
	at com.myapp.app_base.view.AbstractView.enter(AbstractView.java:125)
	at com.vaadin.navigator.Navigator.performNavigateTo(Navigator.java:778)
	at com.vaadin.navigator.Navigator.lambda$navigateTo$9a874efd$1(Navigator.java:702)
	at com.vaadin.navigator.ViewBeforeLeaveEvent.navigate(ViewBeforeLeaveEvent.java:54)
	at com.vaadin.navigator.View.beforeLeave(View.java:79)
	at com.vaadin.navigator.Navigator.runAfterLeaveConfirmation(Navigator.java:730)
	at com.vaadin.navigator.Navigator.navigateTo(Navigator.java:701)
	at com.vaadin.navigator.Navigator.navigateTo(Navigator.java:678)
	at com.github.appreciated.app.layout.builder.providers.AbstractNavigationElementComponentProvider.lambda$setNavigationClickListener$cb8a9285$1(AbstractNavigationElementComponentProvider.java:16)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.vaadin.event.ListenerMethod.receiveEvent(ListenerMethod.java:499)
	... 39 more```

--- a/vaadin-multifileupload-demo/pom.xml
+++ b/vaadin-multifileupload-demo/pom.xml
@@ -20,7 +20,7 @@
         <project.build.java.version>1.8</project.build.java.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <version.vaadin>8.0.3</version.vaadin>
+        <version.vaadin>8.2.0</version.vaadin>
         <widgetset>com.wcs.wcslib.vaadin.widget.multifileupload.DemoWidgetSet</widgetset>
     </properties>
 

--- a/vaadin-multifileupload/pom.xml
+++ b/vaadin-multifileupload/pom.xml
@@ -20,7 +20,7 @@
         <project.build.java.version>1.8</project.build.java.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <version.vaadin>8.0.3</version.vaadin>
+        <version.vaadin>8.2.0</version.vaadin>
         <widgetset>com.wcs.wcslib.vaadin.widget.multifileupload.WidgetSet</widgetset>
 		
         <!-- ZIP Manifest fields -->

